content/docs/lists-and-keys.md

```diff
@@ -260,7 +260,7 @@ const content = posts.map((post) =>
 
 Yukarıdaki örnekte, `Post` bileşeni `props.id` yi okuyabilir, ancak `props.key` i okuyamaz.
 
-### map()'i JSX'e gömmek {#embedding-map-in-jsx}
+### JSX içi map kullanımı {#embedding-map-in-jsx}
```

Comment:
We should use capitalized words. So, this may be better: 

> JSX İçinde map() Kullanımı

What do you think ?

Reply:
It could be better